Example/Backpack NativeUITests/CalendarUITest.swift

```diff
@@ -0,0 +1,88 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import XCTest
+
+class CalendarUITest: BackpackUITestCase {
+    private func goToPreselectedDatesCalendar(_ app: XCUIApplication) {
+        let tablesQuery = app.tables
+        tablesQuery.staticTexts["Calendar"].tap()
+        tablesQuery.staticTexts["With preselected dates"].tap()
```

Comment:
Using this one since it's the story in which it's calendar is not always the current date, but a fixed date, so that tests always run in the same dates